dotCMS/src/main/java/com/dotcms/publisher/business/PublishQueueElementTransformer.java

```diff
@@ -83,6 +84,8 @@ private static Map<String, Object> getMap(final String id, final String type,
             result = getMapForContentlet(id);
         } else if (isLanguage(type)) {
             result = getMapForLanguage(id);
+        } else if ("CAT".equals(id)) {
```

Comment:
this should be a constant

Reply:
done https://github.com/dotCMS/core/pull/23183/commits/49a6bcf7c5d48d87d283555e1283b87a17e75169